### PR TITLE
Fix import error in Python 3.8

### PR DESCRIPTION
**Pull request by Galiley**
Changes:
- Fix python import error in pyload.core ([#3540](https://github.com/pyload/pyload/issues/3540))
- Comment everything related to flask_themes since it has outdated dependencies (last commit in 2015) in pyload.webui
- Fix static url in css in pyload.webui

### CODE REVIEW

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/webui/app/__init__.py [hunk 1]
```diff
@@ -102,7 +104,7 @@ def _configure_logging(cls, app, pycore):
 
     def __new__(cls, pycore, develop=False):
         app = flask.Flask(__name__)
-
+        babel = Babel(app)
```
Comment (vuolter) [inline src/pyload/webui/app/__init__.py]: Codacy Issue found: [Unused variable 'babel'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

File: src/pyload/__init__.py [hunk 2]
```diff
@@ -29,7 +29,15 @@
 USERHOMEDIR = os.path.expanduser("~")
 os.chdir(USERHOMEDIR)
 
-__version__ = pkg_resources.get_distribution(PKGNAME).parsed_version.base_version
+try:
```
Comment (vuolter) [inline src/pyload/__init__.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/__init__.py (hunk 2 above)
Comment (vuolter) [inline src/pyload/__init__.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py [hunk 3]
```diff
@@ -136,28 +129,24 @@ def _init_log(self):
 
         self.log.warning(f"*** Welcome to pyLoad {self.version} ***")
 
-
     def _init_network(self):
-        from .network import request_factory
-        from .network.request_factory import RequestFactory
+        from pyload.core.network import request_factory
```
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py (hunk 3 above)
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py (hunk 3 above)
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py (hunk 3 above)
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py (hunk 3 above)
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py (hunk 3 above)
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/__init__.py (hunk 3 above)
Comment (vuolter) [inline src/pyload/core/__init__.py]: Codacy Issue found: [Unused variable 'request_factory'](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/managers/thread_manager.py [hunk 4]
```diff
@@ -201,9 +201,11 @@ def try_reconnect(self):
         self.pyload.log.debug(f"Old IP: {ip}")
 
         try:
-            reconn = subprocess.Popen(
-                reconnect_script, bufsize=-1, shell=True
-            )  #: , stdout=subprocess.PIPE)
+            reconn = subprocess.Popen(reconnect_script, bufsize=-1)
```
Comment (vuolter) [inline src/pyload/core/managers/thread_manager.py]: Codacy Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/managers/thread_manager.py (hunk 4 above)
Comment (vuolter) [inline src/pyload/core/managers/thread_manager.py]: Codacy Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by Galiley** [comment]
I think it would be better to reimplement pyload for Python 3 by using newer packages and project structure

**Comment by GammaC0de** [comment]
> I think it would be better to reimplement pyload for Python 3 by using newer packages and project structure

What do you mean by that?

**Comment by Galiley** [comment]
@GammaC0de pyload have two version: one for python 2 and one for python 3. We are in 2020 and python 2 is depreciated and as no longer LTS. I think we should focus on improving the python 3 version (pyload-ng) and upgrade the packages in order to make them fit well with newer python version

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881&bid=16794665)

File: src/pyload/core/managers/thread_manager.py [hunk 5]
```diff
@@ -201,9 +201,8 @@ def try_reconnect(self):
         self.pyload.log.debug(f"Old IP: {ip}")
 
         try:
-            reconn = subprocess.Popen(
-                reconnect_script, bufsize=-1, shell=True
-            )  #: , stdout=subprocess.PIPE)
+            #Using shell in subprocess can be unsecure
+            reconn = subprocess.Popen(reconnect_script, bufsize=-1, shell=False)
```
Comment (vuolter) [inline src/pyload/core/managers/thread_manager.py]: Codacy Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)

**Comment by vuolter** [comment]
I think this PR contains several useful fixes. Sadly was closed by the author.

**Pull request closed without merging**